components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -59,25 +61,16 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
     const {TOTAL_USERS: totalUsers, TOTAL_POSTS: totalPosts} = analytics!;
 
     // gather locally available data
-    const {ServiceSettings, DataRetentionSettings} = props.config;
+    const {ServiceSettings, DataRetentionSettings, TeamSettings, GuestAccountsSettings} = props.config;
     const {location} = document;
 
     const sessionLengthWebInDays = ServiceSettings?.SessionLengthWebInDays || -1;
     const dataRetentionEnabled = DataRetentionSettings?.EnableMessageDeletion || DataRetentionSettings?.EnableFileDeletion;
 
     const testURL = () => {
-        const index = data.access.items.findIndex((item) => item.id === 'site-url');
-        if (index >= 0) {
-            const onSuccess = ({status}: any) => {
-                data.access.items[index].status = status === 'OK' ? 'ok' : 'error';
-            };
-            const onError = () => {
-                data.access.items[0].status = 'error';
-            };
-            return testSiteURL(onSuccess, onError, location.origin);
-        }
-
-        return Promise.resolve();
+        const onSuccess = ({status}: any) => setLiveUrlStatus(status === 'OK' ? 'ok' : 'error');
```

Comment:
nit: we should put all these string values either in a enum or simple constant values, as a best practice.